### DISCUSSION TITLE
Events, Menu, Mouse

### DISCUSSION THREAD

**Discussion by KhandamovA** [TurboWarp/extensions#801] [Q&A] [answered]
where can i handle events for my extension? I need an event that updates the menus. And yet, how to get the ID of the block that opened the menu? how to find out the position of the mouse on the block area?

**Comment by KhandamovA** [comment]
I found a way to do it, thanks everyone

**Reply by GarboMuffin** [reply]
perhaps you can elaborate so whoever next wonders this has a solution

**Reply by yuri-kiss** [reply]
yes

**Reply by KhandamovA** [reply]
I will probably have a useful extension, but most likely it will not get into the gallery, does it make sense for me to create a pull request?

**Comment by KhandamovA** [comment, marked as answer]
to get block id i used "document.addEventListener('mousemove', function (event) { event.target ... })"
